splinterd/man/splinterd.1.md

```diff
@@ -445,6 +445,39 @@ node_id = "mynode"
 heartbeat = 60
 ```
 
+The next example demonstrates how to configure GitHub as an OAuth provider for
+REST API authentication, where the placeholder values for the client ID and
+secret would be replaced with actual values for a registered GitHub OAuth app:
+
+```
+$ splinterd --node-id mynode \
+  --oauth-provider github \
+  --oauth-client-id <my-client-id> \
+  --oauth-client-secret <my-client-secret> \
+  --oauth-redirect http://localhost:8080/oauth/callback
+```
+
+The above example assumes that the splinterd REST API is accessible to the web
+browser at the address `http://localhost:8080`; if the REST API has a different
+address, this argument would be changed accordingly, but the `/oauth/callback`
```

Comment:
This complete value could be proxied to something different.  For example, the splinter-ui docker compose files proxy all splinter API's under a URI prefix of `/splinterd/`

Reply:
Added some examples of different REST API addresses and the appropriate redirects.